libraries/hwlib/library/core/hwlib-i2c.hpp

```diff
@@ -494,7 +494,7 @@ class i2c_channel {
 /// - <a href="https://en.wikipedia.org/wiki/I2C">
 ///    I2C Bus</A> (wikipedia)
 ///
-class i2c_bus : protected i2c_primitives {
+class i2c_bus : public i2c_primitives {
```

Comment:
why a i2c change in usart PR?

Reply:
While testing i found that this gave an error " i2c_bus is protected in this context" for i2c_bit_banged

Reply:
allright